UTIF.js

```diff
@@ -1458,7 +1458,7 @@ UTIF.toRGBA8 = function(out, scl)
 		for(var i=0; i<area; i++) {
 			var qi=i<<2, si=i*smpls;  
 			
-			if(window.UDOC) {
+			if(window && window.UDOC) {
```

Comment:
This adjustment does not work in all cases. In my case, only the following check works:

`... if(typeof window === 'object' && window.UDOC) { ... `

Reply:
Your approach seems better. I'll make the change accordingly.